backend/muster-roll-service/src/main/java/org/egov/repository/querybuilder/MusterRollQueryBuilder.java

```diff
@@ -0,0 +1,105 @@
+package org.egov.repository.querybuilder;
+
+import org.apache.commons.lang3.StringUtils;
+import org.egov.web.models.MusterRollSearchCriteria;
+import org.springframework.stereotype.Component;
+
+import java.util.Collection;
+import java.util.List;
+
+@Component
+public class MusterRollQueryBuilder {
+
+    private static final String FETCH_MUSTER_ROLL_QUERY = "SELECT muster.*,"+
```

Comment:
Instead of .*, it is better to write complete list of attributes so that, in furture when schema changes, there are less chances of mismatch.